pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -589,11 +589,16 @@ func init() {
 					Verbs:     sets.NewString("list", "watch", "get"),
 					Resources: sets.NewString("storageclasses"),
 				},
-				// Gluster provisioner
+				// Glusterfs provisioner
 				{
 					Verbs:     sets.NewString("get", "create", "delete"),
 					Resources: sets.NewString("services", "endpoints"),
 				},
+				// Glusterfs & Ceph provisioner
+				{
+					Verbs:     sets.NewString("get"),
+					Resources: sets.NewString("secrets"),
```

Comment:
what's the status on the work to mitigate this exposure?
* only work off of storageclass objects, not PV annotations
* require the targeted secret to be a specific type, indicating intent on the secret-creator's part to have it be used by the provisioner?